tests/conftest.py

```diff
@@ -19,6 +60,16 @@ def t():
     tester.s = tester.Chain()
     return tester
 
+@pytest.fixture(scope="module")
+def chain():
+    s = tester.Chain()
+    s.head_state.gas_limit = 10**9
+    return s
+
```

Comment:
Eventually I think we can remove the `t.s = chain` patter from our test files and either use chain itself or set `t.s = chain` in the `conftest.py`, that can be another PR though.

Reply:
Will do that separately - I think that might break some existing tests that rely on shared state between test cases, so I want to be sure that I get it right.